lib/core/network/ipfs/ipfs_manager.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:ipfs_client_flutter/ipfs_client_flutter.dart';
+
+class IPFSManager {
+  final apiKey = 'apiKey';
+  final apiKeySecret = 'apiKeySecret';
+  final serverUlr = 'https://ipfs.infura.io:5001';
```

Comment:
Ok... guess we need secret - let's load it from firebase?! At least it won't be in the code then..